pinot-query-runtime/src/main/java/org/apache/pinot/query/runtime/operator/AggregateOperator.java

```diff
@@ -83,14 +84,22 @@ public class AggregateOperator extends MultiStageOperator {
   public AggregateOperator(MultiStageOperator inputOperator, DataSchema dataSchema, List<RexExpression> aggCalls,
       List<RexExpression> groupSet, DataSchema inputSchema, long requestId, int stageId) {
     this(inputOperator, dataSchema, aggCalls, groupSet, inputSchema, AggregateOperator.Accumulator.MERGERS, requestId,
-        stageId);
+        stageId, null);
+  }
+
+  public AggregateOperator(MultiStageOperator inputOperator, DataSchema dataSchema, List<RexExpression> aggCalls,
```

Comment:
can we replace the constructor by adding virtual server instead of creating one? which function is still using the constructor without the virtual server address?

Reply:
Mostly test classes

Reply:
oh. i see. in this case lets make it clean -- there's no need to keep additional public constructor just for test purposes.

Reply:
Done.